app/lanes.go

```diff
@@ -0,0 +1,80 @@
+package app
+
+import (
+	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client"
+
+	signerextraction "github.com/skip-mev/block-sdk/adapters/signer_extraction_adapter"
+	"github.com/skip-mev/block-sdk/block/base"
+	defaultlane "github.com/skip-mev/block-sdk/lanes/base"
+	mevlane "github.com/skip-mev/block-sdk/lanes/mev"
+)
+
+const (
+	maxTxPerMEVLane     = 500  // this is the maximum # of bids that will be held in the app-side in-memory mempool
+	maxTxPerDefaultLane = 3000 // all other txs
+)
+
+var (
+	defaultLaneBlockspacePercentage = math.LegacyMustNewDecFromStr("0.90")
+	mevLaneBlockspacePercentage     = math.LegacyMustNewDecFromStr("0.10")
+)
+
+// CreateLanes walks through the process of creating the lanes for the block sdk. In this function
+// we create three separate lanes - MEV, Free, and Default - and then return them.
+func CreateLanes(app *OsmosisApp, txConfig client.TxConfig) (*mevlane.MEVLane, *base.BaseLane) {
+	// 1. Create the signer extractor. This is used to extract the expected signers from
+	// a transaction. Each lane can have a different signer extractor if needed.
+	signerAdapter := signerextraction.NewDefaultAdapter()
+
+	// 2. Create the configurations for each lane. These configurations determine how many
+	// transactions the lane can store, the maximum block space the lane can consume, and
+	// the signer extractor used to extract the expected signers from a transaction.
+
+	// Create a mev configuration that accepts maxTxPerMEVLane transactions and consumes mevLaneBlockspacePercentage of the
+	// block space.
+	mevConfig := base.LaneConfig{
+		Logger:          app.Logger(),
+		TxEncoder:       txConfig.TxEncoder(),
+		TxDecoder:       txConfig.TxDecoder(),
+		MaxBlockSpace:   mevLaneBlockspacePercentage,
+		SignerExtractor: signerAdapter,
+		MaxTxs:          maxTxPerMEVLane,
+	}
+
+	// Create a default configuration that accepts maxTxPerDefaultLane transactions and consumes defaultLaneBlockspacePercentage of the
+	// block space.
+	defaultConfig := base.LaneConfig{
+		Logger:          app.Logger(),
+		TxEncoder:       txConfig.TxEncoder(),
+		TxDecoder:       txConfig.TxDecoder(),
+		MaxBlockSpace:   defaultLaneBlockspacePercentage,
+		SignerExtractor: signerAdapter,
+		MaxTxs:          maxTxPerDefaultLane,
+	}
```

Comment:
So total 3500 txs per block! Wondering if there were reasons these numbers of txs were chose or were arbitrary numbers

Reply:
They are just arbitrary numbers that the SKIP team suggested

Reply:
Its really confusing to me why there is a percentage and a maxTxs number. Our framing should be in terms of block space categories relative to one another rather than a number of transactions.

Reply:
The maxTx is a mempool setting, no? These field names from the block SDK can use some improvement